packages/overlays/src/OverlayMixin.js

```diff
@@ -128,19 +210,30 @@ export const OverlayMixin = dedupeMixin(
       }
 
       /**
-       * @overridable method `_overlayTemplate`
-       * Be aware that the overlay will be placed in a different shadow root.
-       * Therefore, style encapsulation should be provided by the contents of
-       * _overlayTemplate
-       * @return {TemplateResult}
+       * @desc Two options for a Subclasser:
+       * - 1: Define a template in `._overlayTemplate`. In this case the overlay content is
+       * predefined and thus belongs to the web component. Examples: datepicker.
+       * - 2: Define a getter `_overlayContentNode` that returns a node reference to a (content
+       * projected) node. Used when Application Developer is in charge of the content. Examples:
+       * popover, dialog, bottom sheet, dropdown, tooltip, select, combobox etc.
        */
+      get __managesOverlayViaTemplate() {
+        return Boolean(this._overlayTemplate);
+      }
 
-      /**
-       * @overridable method `_defineOverlay`
-       * @desc returns an instance of a (dynamic) overlay controller
-       * @returns {OverlayController}
-       */
-      // eslint-disable-next-line
-      _defineOverlay({ contentNode, invokerNode }) {}
+      __syncOpened() {
+        if (this._opened) {
+          this._overlayCtrl.show();
+        } else {
+          this._overlayCtrl.hide();
+        }
+      }
+
+      __syncPopper() {
```

Comment:
`__syncPopper` is not used anywhere => remove

Reply:
If I recall correctly, I added a TODO to check if still needed, and I'm pretty sure somewhere it is still called but I could be wrong about that :|